doc/source/science_guide/sg_dynamics.rst

```diff
@@ -268,16 +291,66 @@ when :math:`h_u > h_{cu}`.
 
 The maximum seabed stress depends on the weight of the ridge 
 above hydrostatic balance and the value of :math:`k_2`. It is, however, the parameter :math:`k_1` that has the most notable impact on the simulated extent of landfast ice. 
-The value of :math:`k_1` can be changed at runtime using the namelist variable ``k1``. The grounding scheme can be turned on or off using the namelist logical basalstress. 
+The value of :math:`k_1` can be changed at runtime using the namelist variable ``k1``. 
 
-Note that the user must provide a bathymetry field for using this grounding 
-scheme. It is suggested to have a bathymetry field with water depths larger than 
-5 m that represents well shallow water regions such as the Laptev Sea and the 
-East Siberian Sea. To prevent unrealistic grounding, :math:`T_b` is set to zero when :math:`h_{wu}` 
+To prevent unrealistic grounding, :math:`T_b` is set to zero when :math:`h_{wu}` 
 is larger than 30 m. This maximum value is chosen based on observations of large 
 keels in the Arctic Ocean :cite:`Amundrud04`.
 
-   
+Seabed stress based on probabilistic approach
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This new and more sophisticated grounding parameterization computes the seabed stress based
+on the probability of contact between the ice thickness distribution
+(ITD) and the seabed. Multi-thickness category models such as CICE typically use a
+few thickness categories (5-10). This crude representation of the ITD
+does not resolve the tail of the ITD which is crucial for grounding
```

Comment:
```suggestion
does not resolve the tail of the ITD, which is crucial for grounding
```

Reply:
@phil-blain you seem to have good ideas...Right now we choose the method with the integer kseabed. @apcraig  would prefer character strings...any suggestions (should be short I guess)?

Reply:
I think `mean_thickness` and `probabilistic` would be good fits.

Note: you should be able to click the "Accept suggestion" button on the suggestions  I made above, and then they will be committed to your branch (on GitHub). You will then be able to pull these changes locally if you want to tweak the PR further.